mandatory/3-function-output.js

```diff
@@ -1,16 +1,18 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getNumber() {
-  return Math.random() * 10;
-}
+  return Math.random() * 10; 
+} //getNumber will have Math.random() return a random number between 0 (incl.) and 1(excl) and then  is multiplied by 10
```

Comment:
returns random float between 0 and 10

Reply:
Hi. Thank you for your comment. 

Great catch with the word float, attention to detail makes all the difference. Thanks!
Google says it returns a random float between 0 and 1-not 10, perhaps this is a typo?